### DISCUSSION TITLE
Is there any options to add inline bar?

### DISCUSSION THREAD

**Discussion by V-ampire** [aminalaee/sqladmin#309] [Q&A]
Hey, very nice lib!

I wanted to ask is it able to add inlines for related objects like Django inlines (https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#inlinemodeladmin-objects)?

**Comment by aminalaee** [comment]
Hey, no it's not there yet.
In SQLAdmin it would be like Flask-Admin inline_models.

**Reply by aminalaee** [reply]
PRs are welcome though.
